scripts/check_data_injection2.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
There shouldn't be a check_data_injection2.php, so one of these is correct and one of these isn't.

Reply:
maybe a problem for later, though.